css/style.css

```diff
@@ -0,0 +1,67 @@
+/* Styling the body element
+		like body color and margin */
+    #main {
```

Comment:
Please kindly always use the class attribute for multiple elements (do not use ID selector for multiple elements). You are gonna need to put some effort again and fix everything inside the style.css file, mate. 😮‍💨
It's mentioned here. Please kindly follow all these rules.
https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.